src/Tools/dotnet-symbol/Program.cs

```diff
@@ -25,11 +28,13 @@ private struct ServerInfo
         {
             public Uri Uri;
             public string PersonalAccessToken;
+            public bool InternalSymwebServer;
         }
 
         private readonly List<string> InputFilePaths = new();
         private readonly List<string> CacheDirectories = new();
         private readonly List<ServerInfo> SymbolServers = new();
+        private TokenCredential TokenCredential = new DefaultAzureCredential(includeInteractiveCredentials: true);
```

Comment:
why is symbol interactive but not dotnet/dump?

Reply:
dotnet-dump has the option to be interactive with setsymbolserver --interactive. Since we do have customers that script dotnet-dump, I made it opt-in.  With dotnet-symbol, I assume (maybe incorrectly) that it will always be interactive.

Reply:
Should I make dotnet-dump interactive by default and the script users disable it?

Reply:
hmm On linux I expect a good share to be headless and that's going to be the main analyze customer base for this. Thsi sounds good.

Reply:
You are ok with leaving this alone: dotnet-symbol is interactive by default and dotnet-dump isn't.